### ISSUE TITLE
Installation instructions

### ISSUE THREAD

**Issue by heather999** [LSSTDESC/supreme#22]
Hi!
I just added v0.1, using pip, to a DESC python environment (without the stack).  Clearly I haven't used supreme before.. but upon import there is an error concerning missing `esutil`.  I went to install esutil, via `pip install esutil`, but that fails in this Python3.7 miniconda 4.7.12.1 environment.

```
x86_64-conda_cos6-linux-gnu-cc: error trying to exec 'cc1plus': execvp: No such file or directory
    error: command '/global/common/software/lsst/common/miniconda/py3.7-4.7.12.1/envs/desc-dev/bin/x86_64-conda_cos6-linux-gnu-cc' failed with exit status 1
```

What are the typical requirements to install supreme (and esutil)?

**Comment by erykoff** [comment]
I have to figure out why I'm not getting notifications on this repo, I only just saw this.  I will add installation instructions, including the fact that `suprême` can only be run on top of the stack environment.

**Comment by erykoff** [comment]
Closed by #24
